scripts/ci_tests.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#
+# Name::        ci_tests.sh
+# Description:: Use this script to run ci tests of this repository
+# Author::      Salim Afiune Maya (<afiune@lacework.net>)
+#
+set -eou pipefail
+
+readonly project_name=terraform-kubernetes-agent
+
+TEST_CASES=(
+  examples/custom-resource-allocation
+  examples/custom-tolerations
+  examples/default
```

Comment:
this test does exist but is empty. Not sure if we want to add it.

Reply:
Sounds good @afiune!